runtime/lua/vim/lsp/rpc.lua

```diff
@@ -275,7 +275,6 @@ local function create_and_start_client(cmd, cmd_args, handlers, extra_spawn_para
   end
 
   local function send_notification(method, params)
-    local _ = log.debug() and log.debug("rpc.notify", method, params)
```

Comment:
The log output here is sufficient.
https://github.com/neovim/neovim/pull/12468/files#diff-fe036d8f092c13feb2826e6d9daa63bfR267